tests/test_verify.py

```diff
@@ -92,3 +141,14 @@ def test_assert_raises(meth, value, arg):
         meth(value, *arg.args, **opts)
 
     assert opts['msg'] in str(exc.value)
+
+
+def test_expect_and_ensure_are_aliases():
```

Comment:
`test_expect_and_ensure_are_aliases` and `test_assertions_aliases` could be a single, parametrized test like:

```python
@pytest.mark.parametrize('obj,alias', [
    (expect, ensure),
    (Greater, GreaterThan),
    ...snip...
])
def test_aliases(obj, alias):
    assert obj is alias
```